### PR TITLE
Create new CRIB setup script

### PR DESCRIPTION
**Pull request by chainchad**

### CODE REVIEW

File: crib/scripts/cribbit.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ if [[ -z "${DEVSPACE_NAMESPACE}" ]]; then
     exit 1
 fi
 
-# Check if $DEVSPACE_NAMESPACE begins with a crib- prefix and exit 1 if it does not.
+# Bail if $DEVSPACE_NAMESPACE does not begin with a crib- prefix or does not have an orride set.
```
Comment (momentmaker) [inline crib/scripts/cribbit.sh:28]: override?

Comment (chainchad) [reply crib/scripts/cribbit.sh]: 2e66f34608667a6a7a6e98c0240ae99d5ed12620

**Review by momentmaker — APPROVED** [review]

File: crib/scripts/cribbit.sh [hunk 2]
```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+#############################
```
Comment (scheibinger) [inline crib/scripts/cribbit.sh:5]: Nice 😄 That is pretty cool

**Review by scheibinger — DISMISSED** [review]
🚀

**Pull request merged**